app/services/food_store.py

```
@@ -1,24 +1,68 @@
+                else:
+                    aliases[primary] = alias_list
+    except (csv.Error, UnicodeDecodeError):
+        logger.error("Failed to parse aliases CSV '%s'", csv_path, exc_info=True)
+        return {}
+    except FileNotFoundError:
+        logger.error("Aliases CSV not found: %s", csv_path, exc_info=True)
+        return {}
+    except OSError:
+        logger.error("OS error reading aliases CSV '%s'", csv_path, exc_info=True)
+        return {}
+    except Exception as e:  # Unexpected
```

Flake8 (7.3.0)

[error] local variable 'e' is assigned to but never used

(F841)